aea/helpers/ipfs/base.py

```diff
@@ -43,35 +41,13 @@
     from aea.helpers.ipfs.pb.merkledag_pb2 import PBNode  # type: ignore
 
 
-def _dos2unix(file_content: bytes) -> bytes:
-    """
-    Replace occurrences of Windows line terminator CR/LF with only LF.
-
-    :param file_content: the content of the file.
-    :return: the same content but with the line terminator
-    """
-    return re.sub(b"\r\n", b"\n", file_content, flags=re.M)
```

Comment:
This is the logic which caused the inconsistent hash issues when locking and publishing packages, not sure why this was here in the first place but removing this logic fixes the issue and does not cause any new issue AFAIC so can be removed safely